HW1/P2/P2.txt

```diff
@@ -1 +1,3 @@
 # Your discussion here
+1) The partitioning chosen was to uniformly randomly distribute the pixels before the mandelbrot computation. This choice was made based on the first histogram with the default partitioning. In the first histogram, the majority (over 85%) of the pixels had very few iterations. Thus, uniformly mixing the few high iteration pixels along with the majority low iteration pixels intuitively made sense as a more equitable partitioning strategy. The second histogram, while not being uniform in frequency across pixel bins is still much better distributed than the original histogram (it appears to resemble a normal distribution).
```

Comment:
Also, watch your orders of magnitude. Yes, it's not strictly uniform, but the range is much smaller.